lampo-bdk-wallet/src/lib.rs

```diff
@@ -177,11 +199,11 @@ impl WalletManager for BDKWalletManager {
         let txs = wallet
             .list_unspent()
             .map(|tx| Utxo {
-                txid: tx.outpoint.txid.to_hex(),
+                txid: format!("{:x}", tx.outpoint.txid),
```

Comment:
```suggestion
                txid: format!("{:x}", tx.outpoint.txid),
```

You are returning a debug string here, and it is wrong because the txid is used by others to lookup the transaction status, but in this way you are breaking every user